update-http-backend-docs (#1040)
meshcloud/meshcloud-docs: docs/meshstack.building-blocks.meshStack-http-backend.md

- [OpenTofu](https://opentofu.org/docs/language/settings/backends/configuration)
- [Terraform](https://developer.hashicorp.com/terraform/language/backend)

> meshStack accesses building block state with every new run or on deletion. With the built-in http backend, you can store the state file within meshStack. This means it won't be necessary to configure an extra backend within your Terraform/OpenTofu sources. In this case meshStack automatically takes care of the state of the building block. 

Reviewer: ```suggestion
meshStack accesses the building block state with every new run or on deletion. With the built-in http backend, you can store the state file within meshStack. This means it won't be necessary to configure an extra backend within your Terraform/OpenTofu sources. In this case meshStack automatically takes care of the state of the building block. 
```

}
```

> Please note that including a configuration for meshStack's http backend in your source as shown above will not work, because we do not support the configuration of basic authentication values here.

Reviewer: ```suggestion
Please note that including a configuration for meshStack's http backend in your source, as shown above, will not work because basic authentication values are not supported in this context.
```
